_rules/meta-refresh-no-delay-bc659a.md

```diff
@@ -95,7 +95,7 @@ Redirects immediately.
 
 #### Passed Example 2
 
-First valid `<meta http-equiv="refresh">` redirects immediately.
+First valid `<meta http-equiv="refresh">` on the page redirects to a new page immediately.
```

Comment:
We don't want to put HTML inline anymore. Try just using the node name. If necessary use the attributes as a sentence.

```suggestion
First valid `meta` element on the page redirects to a new page immediately.
```